### PR TITLE
Fixed resource URLs for ansible@2.0

### PR DESCRIPTION
**Pull request by fmckeogh**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
If `ansible@2.0` does not build on any macOS version, we should probably remove it altogether. It accounts for 1.2% of ansible installs.

**Comment by ilovezfs** [comment]
```
  bottle do
    cellar :any
    sha256 "f8af604e3a659dec1751261c6993730d88cf7c12707477c5aa7a3c8f9683c68e" => :sierra
    sha256 "1a0cf69816d357429dbdd063e0f29ae407cba9d2afa238702319644ff3425ba5" => :el_capitan
    sha256 "7d878c54552e0e542e10e84adbea7e88d18285b9756c8abaf8d5f38670b17856" => :yosemite
  end
```
I'd say we broke something.

**Comment by ilovezfs** [comment]
`cffi` needs to be bumped, and then it's fine.

**Comment by chocol4te** [comment]
I apologise, I was just going through brew audit issues.

**Comment by fxcoudert** [comment]
Can you update the `cffi` resource to the latest version, as @ilovezfs suggested?

File: Formula/ansible@2.0.rb [hunk 1]
```diff
@@ -83,114 +83,114 @@ class AnsibleAT20 < Formula
   # Install requests with [security]
   #
   resource "cffi" do
-    url "https://pypi.python.org/packages/source/c/cffi/cffi-1.5.2.tar.gz"
+    url "https://files.pythonhosted.org/packages/source/c/cffi/cffi-1.11.tar.gz"
     sha256 "da9bde99872e46f7bb5cff40a9b1cc08406765efafb583c704de108b6cb821dd"
   end
```
Comment (JCount) [inline Formula/ansible@2.0.rb]:
```
resource "cffi" do
    url "https://files.pythonhosted.org/packages/source/c/cffi/cffi-1.10.0.tar.gz"
    sha256 "b3b02911eb1f6ada203b0763ba924234629b51586f72a21faacc638269f4ced5"
  end
```

**Review by JCount — COMMENTED** [review]
This is breaking because the most recent version released outside of PyPy 5.8.0 is 1.10.0.

File: Formula/ansible@2.0.rb (hunk 1 above)
Comment (JCount) [inline Formula/ansible@2.0.rb]: The way the "latest" version, v1.11 isn't released yet anywhere, it is `(NOT RELEASED SO FAR)` is a little odd...

**Comment by chocol4te** [comment]
Fixed after a few silly commits.

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by JCount** [comment]
@chocol4te I'm sorry that this PR stalled for no good reason. When I looked at the current conflicts this PR had for `conjure-up` I felt it made the most sense for me to simply open a new PR based on the updated `requirements.txt` file in the `2.2.2` release.

To try and still give you at least some credit for your contributions I've rebased out the `conjure-up` commit and force-pushed the branch. Once it passes CI, I'll merge the `ansible@2.0` commits.

**Comment by JCount** [comment]
@chocol4te Thank you for your contribution to Homebrew!!! 🎊 It really is nice to see new people contributing to the community. 😄

**Comment by chocol4te** [comment]
@JCount That's really kind of you say, thanks!

**Pull request closed without merging**